WalletWasabi.Tests/UnitTests/SmartLabelTests.cs

```diff
@@ -79,8 +79,8 @@ public void LabelEqualityTests()
 			var label3 = new SmartLabel("bar, buz");
 			Assert.NotEqual(label, label3);
 
-			SmartLabel label4 = null;
-			SmartLabel label5 = null;
+			SmartLabel label4 = null!;
+			SmartLabel label5 = null!;
```

Comment:
Why not make these variables nullable instead? The test passes also that way.

Reply:
You are right. Thank you